templates/common/.devcontainer/devcontainer.json/nodejs/terraform/devcontainer.json

```diff
@@ -41,7 +37,7 @@
         3100
     ],
     "postCreateCommand": "",
-    "remoteUser": "vscode",
+    "remoteUser": "node",
```

Comment:
Please make sure to regression test all devcontainers in remote-containers vscode AND codespaces.